lib/CMakeLists.txt

```diff
@@ -27,6 +27,4 @@ set(lib-objs $<TARGET_OBJECTS:aggregator-objs>)
 # Visible to parent directory
 set(lib-objs ${lib-objs} PARENT_SCOPE)
 
-add_subdirectory(ml)
```

Comment:
There is no cpp under `ml` dir to be compiled as the previous `vector_linalg.cpp` is substituted by Vector class.